chart/values.yaml

```diff
@@ -1007,6 +1007,7 @@ dags:
     # interval between git sync attempts in seconds
     wait: 60
     containerName: git-sync
+    # TODO: Should this be 65534?
     uid: 65533
```

Comment:
Itt should be able 65533. See: https://github.com/kubernetes/git-sync/blob/b0ac361b2cf1a1c278bfd5e0cd0368131ddabd30/Dockerfile.in#L62